packages/firestore/src/api/database.ts

```diff
@@ -202,6 +202,10 @@ class FirestoreConfig {
   persistence: boolean;
 }
 
+// TODO(multi-tab): Replace with Firestore.PersistenceSettings
```

Comment:
multitab (or else pinky swear that you'll grep for "TODO(multi" before we declare multi-tab done).

Reply:
Fixed